mk/cargo.sh

```diff
@@ -251,16 +251,16 @@ cargo "$@"
 if [ -n "${RING_COVERAGE-}" ]; then
   # Keep in sync with check-symbol-prefixes.sh.
   # Use the host target-libdir, not the target target-libdir.
-  llvm_root="$(rustc +${toolchain} --print target-libdir)/../bin"
+  llvm_root="$(rustc +"${toolchain}" --print target-libdir)/../bin"
 
-  while read executable; do
+  while read -r executable; do
     basename=$(basename "$executable")
-    ${llvm_root}/llvm-profdata merge -sparse "$coverage_dir/$basename.profraw" -o "$coverage_dir/$basename.profdata"
+    "${llvm_root}"/llvm-profdata merge -sparse "$coverage_dir/$basename.profraw" -o "$coverage_dir/$basename.profdata"
```

Comment:
It looks weird to be inconsistent with having the double quotes only around the variable usage in `"${llvm_root}"/llvm-profdata` but then less tight in `"$coverage_dir/$basename.profraw"` and `"$coverage_dir/$basename.profdata"`. Maybe I'm missing something, but some more consistency would be nice.

Reply:
Resolved by 31dc1e6eb4badf6ff3f09310d729d3784fc60f39.

Though I was thinking about normalizing the usage of `${example}` vs `$example` using the following rules:
* Use `$example` when the variable is not being without additional characters or variable modifiers (e.g. `${example:-default_value}`
* Use `${example}` otherwise, e.g. `+${example}`, `my/path/${example}` to make it more (visually) obvious that a string is being built from mixing strings with variables.

@briansmith Are you ok with this?